lib/rdf.js

```diff
@@ -64,6 +75,15 @@ function createRDF(manifest) {
     delete jetpackMeta["em:icon64URL"];
   }
 
+  if (needsBootstrapJS) {
```

Comment:
Huh, I didn't know this was done in `bootstrap.js`. In any case I suppose we can trust developers using custom `bootstrap.js` with this. Can you please add a very short comment above that line so that people like me understand what's going on? :sweat_smile: